### PR TITLE
docs: move contributing to own page

### PR DESCRIPTION
**Pull request by eoaksnes**
## Why is this pull request needed?

* Contributing is separate concept from Docs so we move it out to own page

## What does this pull request change?

* Move contributing out from Docs

## Issues related to this change:

image

### CODE REVIEW

**Comment by eoaksnes** [comment]
Må vist fikse noen linker ser jeg.... on it

File: documentation/docs/contribute/01-contribution-guide.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Contributing
+# How to start contributing
```
Comment (ingeridhellen) [inline documentation/docs/contribute/01-contribution-guide.md:1]: There is a mismatch between file name and header that could be confusing

**Review by sebastianvitterso — APPROVED** [review]
tenor-111392115

File: documentation/docusaurus.config.js [hunk 2]
```diff
@@ -71,18 +64,18 @@ async function createConfig() {
           },
           items: [
             {
-              type: 'doc',
-              docId: 'introduction',
+              type: 'docSidebar',
+              sidebarId: 'about',
               position: 'left',
               label: 'Docs',
```
Comment (ingeridhellen) [inline documentation/docusaurus.config.js:70]: The mismatch between label, sidebarId and folder name could be confusing

File: documentation/docusaurus.config.js [hunk 3]
```diff
@@ -98,11 +91,11 @@ async function createConfig() {
               items: [
                 {
                   label: 'Docs',
-                  to: '/docs/introduction',
+                  to: '/docs/about/introduction',
```
Comment (ingeridhellen) [inline documentation/docusaurus.config.js:94]: Maybe add contribute to footer as well?

File: documentation/docusaurus.config.js (hunk 2 above)
Comment (eoaksnes) [reply documentation/docusaurus.config.js:70]: @ingeridhellen  Not sure how to best solve this, either we need to have /docs/docs and Docs or use /docs/about and About. Any input?

File: documentation/docusaurus.config.js (hunk 3 above)
Comment (eoaksnes) [reply documentation/docusaurus.config.js:94]: added

File: documentation/docs/contribute/01-contribution-guide.md (hunk 1 above)
Comment (eoaksnes) [reply documentation/docs/contribute/01-contribution-guide.md]: fixed

File: documentation/docusaurus.config.js (hunk 2 above)
Comment (eoaksnes) [reply documentation/docusaurus.config.js:70]: we do this another PR if required

**Pull request merged**